doofinder.php

```diff
@@ -31,9 +31,11 @@ class Doofinder extends Module
     public $ps_layered_full_tree = true;
     public $searchBanner = false;
 
+    // Feel free to change this value to your own local env or ngrok
+    const DOOMANAGER_URL = 'https://admin.doofinder.com';
```

Comment:
eu1 and us1 regions are compatible with this url?

Reply:
In the commit where this part was deleted it was using this one, but I'll try to research about it:
![image](https://github.com/doofinder/doofinder-prestashop/assets/128705267/eccd5bf2-c823-4ef0-91b6-4d6abe1089ac)

Reply:
Thank you for the feedback :smiley:

Reply:
I was able to open both of them without any issue

Reply:
Explanation of why this works:

When a user accesses, he is redirected to a region depending on the latency he has, that is, when he is accessing from Spain, he is redirected to the EU region, which is the one with the lowest response time.

The URL that you have configured is simply to check if the client's Prestashop can connect with Doofinder regardless of the region. In the end, what it does is simply a ping to the Doofinder URL, so we don't care if it goes to one region or another.

Reply:
Thanks @davidmolinacano for the explanation

Reply:
@ivan-doofinder has explained it to me with this level of detail, so he deserves the credit :clap: